setup.py

```diff
@@ -41,13 +41,12 @@
         install_requires=[
             "inflection>=0.2",
             "Markdown>=2.4",
-            "jsonschema>=2.3,<4.18",
+            "jsonschema>=4.18",
             "six>=1.5.2",
         ],
         cmdclass=versioneer.get_cmdclass(),
         classifiers=[
             "Programming Language :: Python :: 3",
-            "Programming Language :: Python :: 3.7",
```

Comment:
In addition to updating the classifiers, I suggest also passing `python_requires=">=3.8"` to `setup`, though I *think* everything will work out okay even if this is not done.

This project has not been using the `python_requires` argument before. But classifiers are advisory in nature, and I believe `pip` will never treat them as constraints that need to be solved when resolving specific dependency versions.

However, everything *should* be okay even without it, because while classifiers aren't constraints, dependency version ranges in `install_requires` are. Because `jsonschema` 4.18 itself constrains the Python version to be `>=3.8`, I *think* `pip` on Python 3.7 will correctly manage to select an appropriate lower version of both `python-jsonschema-objects` and `jsonschema`.